docs/guides/running-grpc.md

```diff
@@ -0,0 +1,45 @@
+## Installation
+
+Please follow the [deployment](../deployment) documentation to install
+Kong Ingress Controller onto your Kubernetes cluster.
+
+## Testing connectivity to Kong
+
+This guide assumes that `PROXY_IP` environment variable is
+set to contain the IP address or URL pointing to Kong.
+If you've not done so, please follow one of the
+[deployment guides](../deployment) to configure this environment variable.
+
+If everything is setup correctly, making a request to Kong should return
+HTTP 404 Not Found.
+
+```bash
+$ curl -i $PROXY_IP
+HTTP/1.1 404 Not Found
+Date: Fri, 21 Jun 2019 17:01:07 GMT
+Content-Type: application/json; charset=utf-8
+Connection: keep-alive
+Content-Length: 48
+Server: kong/1.2.1
+
+{"message":"no Route matched with those values"}
+```
+
+This is expected as Kong does not yet know how to proxy the request.
+
+#### Running GRPC
+
+
+1. Add a grpc deployment and service
+
+`kubectl apply -f ../../deploy/manifests/sample-apps/grpc.yaml`
```

Comment:
These need to be absolute Github links. We do not expect that the user will clone this repository.

Reply:
Any objection to my using bit.ly links to point to aspirational paths: e.g., https://raw.githubusercontent.com/Kong/kubernetes-ingress-controller/master/deploy/manifests/sample-apps/sample-grpc-ingress.yaml?